### ISSUE TITLE
Pause July Archive-It collection

### ISSUE THREAD

**Issue by edsu** [sul-dlss/was-registrar-app#482]
To ensure we don't need to run indexing again it would be useful to put our July Archive-It data collection temporarily on hold until the current indexing job is finished and we are live on pywb.

**Comment by justinlittman** [comment]
Just need to comment this out and deploy: https://github.com/sul-dlss/was-registrar-app/blob/main/config/schedule.rb#L14-L16

**Comment by justinlittman** [comment]
Collecting is disabled on prod; once indexing is done, we need to re-enable.

**Comment by edsu** [comment]
I created #484 for re-enabling. This change has been deployed to prod, so collection is paused for now.
